File: temp/find_quotes.js
const fs = require('fs');
const path = '/d/新课开发/金融/6、保险增员面谈与代理人团队裂变——从招人到留人/授课PPT/slides';

const failed = [9,10,11,12,13,14,32,34,35,37,39,40,41,50,58,61,68,73,79];

failed.forEach(n => {
  const fname = path + '/slide-' + n + '.js';
  if (!fs.existsSync(fname)) { console.log('NOT FOUND:', fname); return; }
  const content = fs.readFileSync(fname, 'utf8');
  const lines = content.split('\n');
  const problematic = [];
  lines.forEach((line, i) => {
    if (line.includes('‘') || line.includes('’') || 
        line.includes('“') || line.includes('”')) {
      problematic.push(i+1);
    }
  });
  if (problematic.length > 0) {
    console.log('slide-' + n + '.js:', problematic.join(','), 'lines have curly quotes');
  } else {
    console.log('slide-' + n + '.js: no curly quotes');
  }
});

File: temp/find_quotes.py
import os
import re

slides_dir = r"D:/新课开发/企业大学/对内/1.企业大学重生：从内训中心到知识资产与智能体孵化枢纽的角色转型/PPT演示文稿/slides"

# Pattern to find Chinese curly quotes " " inside JS strings
pattern = re.compile(r'"([^"]*)"([^"]*)"([^"]*)"')

for f in sorted(os.listdir(slides_dir)):
    if f.startswith('slide-') and f.endswith('.js'):
        path = os.path.join(slides_dir, f)
        with open(path, 'r', encoding='utf-8') as file:
            content = file.read()

        # Look for lines with potential issues
        lines = content.split('\n')
        for i, line in enumerate(lines, 1):
            # Skip comments
            if line.strip().startswith('//'):
                continue
            # Look for Chinese opening quote followed by Chinese text and Chinese closing quote
            if '"' in line and '"' in line:
                # Check if it's a problematic pattern
                if re.search(r'"[^"]*"[^"]*"[^"]*"', line):
                    print(f"{f}:{i}: {line.strip()[:100]}")